src/main/java/fi/aalto/cs/apluscourses/intellij/services/PluginSettings.java

```diff
@@ -46,8 +56,27 @@ public MainViewModel getMainViewModel(@Nullable Project project) {
     return mainViewModels.computeIfAbsent(project, this::createNewMainViewModel);
   }
 
+  @NotNull
   private MainViewModel createNewMainViewModel(@NotNull Project project) {
     ProjectManager.getInstance().addProjectManagerListener(project, projectManagerListener);
     return new MainViewModel();
   }
+
+  public static boolean isShowReplConfigurationDialog() {
```

Comment:
Nit: I would personally prefer something like `shouldShowReplConfigurationDialog`

Reply:
that's what you get when generating getter/setter for boolean with IJ, not optimal. I'll take into account in the future.